index.html

```diff
@@ -0,0 +1,3 @@
+teste 1
```

Comment:
Esta linha ficou estranha sugiro modificar para:
```suggestion
teste 1 final
```
o que vc acha?

Reply:
Blz vou fazer modificação Grato